crates/astria-conductor/src/executor/client.rs

```diff
@@ -68,7 +71,14 @@ impl Client {
         transactions: Vec<Vec<u8>>,
         timestamp: Timestamp,
     ) -> eyre::Result<Block> {
-        let transactions = transactions.into_iter().map(Bytes::from).collect();
+        use prost::Message;
+
+        let transactions = transactions
+            .into_iter()
+            .map(|tx| RollupData::decode(tx.as_slice()))
```

Comment:
Now that I see this bit I am wondering if making `RollupData` an opaque field inside `RollupTransactions` was the right call.

There is now a lot of wrapping/unwrapping going on and I am not quite sure if that's good.

Reply:
yeah, my intention was that the rollup node decodes the bytes inside `RollupTransactions`, and we consider conductor part of a rollup node, so this is intended. the decoding will happen at one place or another, if we don't put it here, it will be when conductor pulls the sequencer block from sequencer/DA